snuba/query/parser/expressions.py

```diff
@@ -1,8 +1,8 @@
 from dataclasses import replace
 from typing import List, Tuple, Union
 
-from parsimonious.grammar import Grammar  # type: ignore
-from parsimonious.nodes import Node, NodeVisitor  # type: ignore
+from parsimonious.grammar import Grammar
+from parsimonious.nodes import Node, NodeVisitor
```

Comment:
These don't seem to have made it into the configuration file.

Reply:
Thanks